ocw_data_parser/ocw_data_parser.py

```diff
@@ -528,6 +528,8 @@ def generate_parsed_json(self):
         foreign_media = gather_foreign_media(self.jsons)
         self.large_media_links = foreign_media
 
+        is_update_of_list = self.jsons[0].get("is_update_of", [])
+        is_update_of = is_update_of_list[0] if is_update_of_list else None
```

Comment:
Does why not save the full list in the parsed json?

Reply:
From the issue #141 it seems there is only one item in the list so there isn't a need to preserve the list here